index.html

```diff
@@ -2,8 +2,12 @@
 <html lang="en" style="scrollbar-gutter: stable">
   <head>
     <meta charset="UTF-8" />
+    <script src="wasm_exec.js"></script>
```

Comment:
Can't you inject this in the code with lazy loading instead of adding it here in plain?

Reply:
Couldn't find a way. All the examples I found online are like this

Reply:
Solved it now